### PR TITLE
Single intercomparisons

### PR DESCRIPTION
**Pull request by ivanzvonkov**
- Adds single intercomparisons for Togo and Senegal in separate notebooks
- Adds code to generate GEE script for ensemble covermaps

### CODE REVIEW

File: src/compare_covermaps.py [hunk 1]
```diff
@@ -582,7 +624,7 @@ def generate_report(dataset_name: str, country: str, true, pred) -> pd.DataFrame
                 ee.Image("users/eutzschn/Ethiopia_Bure_Jimma_2020_v1"),
                 ee.Image("users/izvonkov/Ethiopia_Bure_Jimma_2019_v1"),
             ]
-        ),
+        )""",
```
Comment (ivanzvonkov) [inline src/compare_covermaps.py:627]: Decided to push the earthengine collections as strings so they can be executed on GEE in Javascript easily

**Comment by hannah-rae** [comment]
Looks great!

File: src/compare_covermaps.py (hunk 1 above)
Comment (hannah-rae) [reply src/compare_covermaps.py:627]: LGTM. I like the P/R/F1 plot and sorting by F1 score is helpful.

**Review by hannah-rae — APPROVED** [review]

**Comment by ivanzvonkov** [comment]
Added Zambia intercomparison in the same format in this PR

**Pull request merged**